res/css/photon/message-bar.css

```diff
@@ -5,53 +5,66 @@
 /* See https://design.firefox.com/photon/components/message-bars.html */
 .photon-message-bar {
   display: flex;
-  width: 100%;
```

Comment:
I removed this because this was in the way of using it in a flex container to make it adjust to its content.
In existing usages it was a noop because divs always take the full width.